addons/web/static/src/core/assets.js

```diff
@@ -108,23 +119,58 @@ export const loadBundleTemplates = memoize(async function loadBundleTemplates(na
     return processTemplates(templates);
 });
 
+
+/**
+ * Loads the content definition of a bundle
+ * @param {string} name the name of the bundle as declared in the manifest.
+ * @returns {Promise<BundleInfo[]>} A promise of the content definition of the bundle
+ * @private
+ */
+const _loadBundle = memoize(async function (bundleName) {
+    const bundleURL = new URL(`/web/bundle/${bundleName}`, window.location.origin);
+    const request = await browser.fetch(bundleURL.href, {
+        method: "get",
+        header: {
+            "Content-Type": "application/json"
+        },
+        param: JSON.stringify({}),
```

Comment:
probably left-over from trying to use the route as `type=json`, `Content-Type` doesn't make sense for `GET` requests (no body), `param` is not a standard field for the fetch `init` parameter (probably ignored), and `get` is already the default method for fetch.